uisdk/src/main/AndroidManifest.xml

```diff
@@ -16,7 +16,8 @@
             android:name="com.braintreepayments.api.BraintreeBrowserSwitchActivity"
             android:label="@string/kh_uisdk_title_activity_threedsecure"
             android:launchMode="singleTask"
-            android:screenOrientation="portrait">
+            android:screenOrientation="portrait"
+            android:exported="true">
```

Comment:
Shouldn't the android:exported be applied to all activities that might be launched by a DP ?

Reply:
We can make a release candidate and use it in the sample app and see if there is any other issue with Android 12

Reply:
i did try it on farechoice on local, seemed to work ok

Reply:
You probably need to update the Farechoice to use Android 12 as well as update the UI SDK version